R/sendmailR.r

```diff
@@ -141,6 +141,39 @@
   send_command("QUIT", 221)
 }
 
+.smtp_submit_mail_curl <- function(smtp_server, headers, curlopts = list(), 
+                                   msg, verbose=FALSE, use_ssl = "force") {
+
+  stopifnot(is.character(headers$From))
```

Comment:
Call `library("curl")` here to only load/attach the `curl` package when needed. That way, users can continue to use the "old" code without a new dependency.